src/openms/source/SYSTEM/UpdateCheck.cpp

```diff
@@ -73,26 +73,38 @@ namespace OpenMS
     platform = "Win";
 #elif __APPLE__
     platform = "Mac";
-#else
+#elif __linux__
     platform = "Linux";
+#elif __unix__
+    platform = "Unix";
+#else
+    platform = "unknown";
 #endif
 
     // write to tmp + userid folder
 
     // e.g.: OpenMS_Default_Win_64_FeatureFinderCentroided_2.0.0
     String tool_version_string;
+    String config_path;
+    //Comply with https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html on unix identifying systems
+    #ifdef __unix__
+    if(getenv("XDG_CONFIG_HOME"))
+      config_path = String(getenv("XDG_CONFIG_HOME")) + "/.config/OpenMS";
```

Comment:
from how I read the docs, `$XDG_CONFIG_HOME` already includes `..config/`, or am I misreading that?

Reply:
Reading the doc again, I believe you are correct. I'll change that.